src/main/java/com/nvidia/spark/rapids/jni/JSONUtils.java

```diff
@@ -166,80 +166,78 @@ public static ColumnVector extractRawMapFromJsonString(ColumnView input, JSONOpt
   }
 
   /**
-   * Extract key-value pairs for each output map from the given json strings. This method is
-   * similar to {@link #extractRawMapFromJsonString(ColumnView, JSONOptions)} but is deprecated.
-   *
-   * @deprecated This method is deprecated since it does not have parameters to control various
-   * JSON reader behaviors.
+   * Parse a JSON string into a struct column following by the given data schema.
+   * <p/>
+   * Many JSON options in the given {@code opts} parameter are ignored from passing down to the
+   * native code. That is because these options are hard-coded with the same values in both the
+   * plugin code and native code. Specifically:<br>
+   * - {@code RecoverWithNull: true}<br>
+   * - {@code MixedTypesAsStrings: true}<br>
+   * - {@code NormalizeWhitespace: true}<br>
+   * - {@code KeepQuotes: true}<br>
+   * - {@code StrictValidation: true}<br>
+   * - {@code Experimental: true}
    *
    * @param input The input strings column in which each row specifies a json object
-   * @return A map column (i.e., a column of type {@code List<Struct<String,String>>}) in
-   *         which the key-value pairs are extracted directly from the input json strings
+   * @param schema The schema of the output struct column
+   * @param opts The options for parsing JSON strings
+   * @param isUSLocale Whether the current local is US locale, used when converting strings to
+   *        decimal types
+   * @return A struct column in which each row is parsed from the corresponding json string
    */
```

Comment:
I assume that this version of extractRawMapFromJsonString was just not used so it is okay for us to delete it.